### PR TITLE
Fixes #26162 .

### LINKED ISSUE #26162: [New Feature] Support PostgreSQL pg_catalog.pg_stat_database_conflicts view empty query
# Background
This task is a subtask of issue #24662.

Hi community.
ShardingSphere is committed to providing a multi-source heterogeneous, enhanced database platform and further building an ecosystem around the upper layer of the platform.
So ShardingSphere needs to be compatible with the inquiries of the system database of various types of databases, such as MySQL, PostgreSQL, and OpenGauss.

So we want to mock all system database tables and views query, return to the empty result.

# Tasks
In this issue, we will implement the empty query of the following system tables/views:
- [ ] pg_stat_database_conflicts
- [ ] pg_stat_gssapi
- [ ] pg_stat_progress_analyze
- [ ] pg_stat_progress_basebackup
- [ ] pg_stat_progress_cluster

# Refer PR
You can refer to the following similar PR to complete the task.
MySQL: #25076, #25256, PostgreSQL: #25752

### PR DESCRIPTION
**Pull request by Master-sudo-doct**
Fixes #26162 .

Changes proposed in this pull request:
  -

---

Before committing this PR, I'm sure that I have checked the following options:
- [x] My code follows the [code of conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/) of this project.
- [x] I have self-reviewed the commit code.
- [x] I have (or in comment I request) added corresponding labels for the pull request.
- [x] I have passed maven check locally : `./mvnw clean install -B -T1C -Dmaven.javadoc.skip -Dmaven.jacoco.skip -e`.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added corresponding unit tests for my changes.

### CODE REVIEW

File: infra/common/src/main/java/org/apache/shardingsphere/infra/metadata/database/schema/builder/SystemSchemaBuilderRule.java [hunk 1]
```diff
@@ -131,7 +132,7 @@ public enum SystemSchemaBuilderRule {
      * Value of builder rule.
      *
      * @param databaseType database type
-     * @param schema schema
+     * @param schema       schema
```
Comment (tuichenchuxin) [inline infra/common/src/main/java/org/apache/shardingsphere/infra/metadata/database/schema/builder/SystemSchemaBuilderRule.java:135]: This appears to be an inadvertent modification

File: infra/common/src/main/java/org/apache/shardingsphere/infra/metadata/database/schema/builder/SystemSchemaBuilderRule.java [hunk 2]
```diff
@@ -144,7 +145,7 @@ public static SystemSchemaBuilderRule valueOf(final String databaseType, final S
     /**
      * Judge whether current table is system table or not.
      *
-     * @param schema schema
+     * @param schema    schema
```
Comment (tuichenchuxin) [inline infra/common/src/main/java/org/apache/shardingsphere/infra/metadata/database/schema/builder/SystemSchemaBuilderRule.java:148]: This appears to be an inadvertent modification

**Review by tuichenchuxin — CHANGES_REQUESTED** [review]

File: infra/common/src/main/java/org/apache/shardingsphere/infra/metadata/database/schema/builder/SystemSchemaBuilderRule.java (hunk 2 above)
Comment (Master-sudo-doct) [reply infra/common/src/main/java/org/apache/shardingsphere/infra/metadata/database/schema/builder/SystemSchemaBuilderRule.java:148]: It seems like so, should I make a new commit to rectify it?

Comment (tuichenchuxin) [reply infra/common/src/main/java/org/apache/shardingsphere/infra/metadata/database/schema/builder/SystemSchemaBuilderRule.java:148]: Just modify it directly on the branch you submitted

Comment (Master-sudo-doct) [reply infra/common/src/main/java/org/apache/shardingsphere/infra/metadata/database/schema/builder/SystemSchemaBuilderRule.java]: done

**Review by tuichenchuxin — APPROVED** [review]

**Pull request merged**